src/devices/Interop/Unix/Libc/Interop.videodev2.struct.cs

```diff
@@ -132,17 +132,43 @@ internal enum v4l2_colorspace : uint
     V4L2_COLORSPACE_RAW = 11,
 }
 
-[StructLayout(LayoutKind.Sequential)]
+[StructLayout(LayoutKind.Explicit)]
 internal struct v4l2_pix_format
 {
+    [FieldOffset(0)]
     public uint width;
+    [FieldOffset(4)]
     public uint height;
+    [FieldOffset(8)]
     public PixelFormat pixelformat;
+    [FieldOffset(12)]
     public v4l2_field field;
+    [FieldOffset(16)]
     public uint bytesperline;
+    [FieldOffset(20)]
     public uint sizeimage;
+    [FieldOffset(24)]
     public v4l2_colorspace colorspace;
+    [FieldOffset(28)]
     public uint priv;
+    [FieldOffset(32)]
+    public uint flags;
+
+    // TODO: Implement v4l2_ycbcr_encoding enum
```

Comment:
remove the TODOs, there is no point implementing that unless we plan to do something with it.